### PR TITLE
Faux Loyalty Implants

### PR DESCRIPTION
**Pull request by giraffism**
This PR adds in a new uplink traitor item, the faux loyalty implant for the price of 2 TC. This implant broadcasts the same green square from loyalty implants for security HUD's users to see. It does not influence the loyalty or alignment of the user. The implant can be activated and deactivated via a random emote.
The implant appears to be a regular loyalty implant until it is removed and is examined.
:cl:
 * rscadd: Adds a new faux loyalty implant to the traitor uplink. Used to broadcast a fake loyalty implant signal for security HUD's to see

### CODE REVIEW

**Comment by giraffism** [comment]
the map check failed, even thought this isnt even a mapping PR

**Comment by ShiftyRail** [comment]
That's because it's conflicting.

**Comment by ShiftyRail** [comment]
I don't really see what niche it fills. Revs can't have it naturally, cult can't have it naturally, and if they did, it would completely change those gamemodes (and would probably be a very bad idea).

Greytide implants are still active on an loyalty-implanted slave. You need to physically remove the implant to de-convert the guy. So the only usage for a regular traitor is to spoof their loyalty status. You could then greytide someone while fooling everyone.
But that seems like a very limited use.

In other words, this item does basically nothing for the regular traitor, but has the potential to completely destroy rounds if a traitor-cult or a traitor-rev happens. 🤔

**Comment by BomberBro** [comment]
Possibility for this to exist during cult or revs make for something truly cancerous. Like it or not, HUDs are a part of team antag modes and making those HUDs suddenly not matter makes for a more unfun experience for everybody. Cult would be a lot more annoying if a traitor could get his own cult HUD then murder people without them realizing (although deadchat might enjoy it)

**Comment by dunsel-hordon** [comment]
I don't really see a problem with this except the cost. For being able to enable sneak mode for a rev or cultist or even a thrall / vampire it should cost at least 7 crystals for each implant instead of 2.

**Comment by BomberBro** [comment]
it's less sneak mode and more "the sole way to tell who's a rev/cultist or not for non-revs/cultists no longer works". The gamemodes were built around this fact

**Comment by giraffism** [comment]
I decided to add this because security is starting to rely more and more on loyalty implants to be able to metacheck and differentiate confirmed non antagonists with everyone else, this was intended to bring back paranoia, however the concerns about revs is valid and i will consider making it impossible for the implant to function in a revolutionary

**Comment by ghost** [comment]
You're neglecting to consider how this affects things from an administrative perspective and how situations can be judged when implants can no longer be trusted. This will inevitably lead to a situation where a gamer HoS murders a bunch of implantees and tries to claim they weren't real implants because they were acting suspicious.

I'm all for chaos but the entire point of implants is to be a metacheck. If you need to avoid getting implanted then you're supposed to not get caught acting suspicious, and then talk your way out of it if that fails. If security blows your brains out on the spot because you turned down an implant and it's not something like code red cult then you can totally ahelp it because that's on the player and not implants as a mechanic.

**Comment by giraffism** [comment]
and dunsel I made the price low because of how much of a niche item this is that it might not ever been seen if the price was too high, however thinking about it I am questioning my own stance on it

**Comment by BomberBro** [comment]
>I decided to add this because security is starting to rely more and more on loyalty implants to be able to metacheck and differentiate confirmed non antagonists with everyone else, this was intended to bring back paranoia

This sort of paranoia sucks for anybody except the antagonist because it's incredibly forced. Security implanting people for 2 hours after the four cultists get splattered because they have no idea if all the cultists are really dead isn't fun for anybody even though it's "paranoia". The fact that you bring up the problems with revs but not cult worries me a bit too, this shouldn't work for any team antag.

**Comment by giraffism** [comment]
due to background issues with github this PR will be closed and re opened at a later date, continue discussing any issues with the PR because legitimate concerns were brought up that I will address
